app/models/matchup.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: matchups
+#
+#  id          :bigint           not null, primary key
+#  winner_id   :bigint
+#  position    :integer
+#  bracket_id  :bigint           not null
+#  option_a_id :bigint
+#  option_b_id :bigint
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+class Matchup < ApplicationRecord
+  belongs_to :bracket
+  belongs_to :winner, class_name: "Entry"
+  belongs_to :option_a, class_name: "Entry"
+  belongs_to :option_b, class_name: "Entry"
+
+  scope :untouched, -> { where(option_a.empty? && option_b.empty?) }
+  scope :unplayed, -> { where(winner.empty?) }
```

Comment:
Do these scopes work? My assumption is that 

```rb
option_a.empty? && option_b.empty?
```
will evaluate to either `true` or `false` and so the scope becomes
```rb
scope :untouched, -> { where(true) }
```
which isn't a valid query.

Perhaps you mean something like

```suggestion
  scope :untouched, -> { where(option_a: nil, option_b: nil) }
  scope :unplayed, -> { where(winner: nil) }
```

Reply:
Yes. That is what I was meaning to do. I hadn't tested them yet, but had planned on doing that when the app was starting to come together a bit more.